mirage-profile.opam

```diff
@@ -13,12 +13,10 @@ build: [
 ]
 
 depends: [
-  "ocamlfind" {build}
   "jbuilder"  {build & >="1.0+beta9"}
-  "cstruct" {>= "1.9.0"}
+  "cstruct" {>= "3.0.0"}
   "ppx_cstruct" {build}
   "ocplib-endian"
   "lwt"
-  "topkg" {build}
 ]
-available: [ ocaml-version >= "4.02.0" ]
+available: [ ocaml-version >= "4.03.0" ]
```

Comment:
Just bumping up the baseline supported version to reduce our testing matrix, in line with other MirageOS3 libraries. Two active versions of OCaml is enough, and there are subtle silent failures with OCaml 4.02.3 when using attributes that we might as well avoid.